frontend/pages/api/tinyUrl.ts

```diff
@@ -0,0 +1,35 @@
+import { NextApiRequest, NextApiResponse } from "next";
+
+const API_KEY = process.env.NEXT_PUBLIC_TINY_URL_API_KEY;
+
+export default async function createTinyUrl(req: NextApiRequest, res: NextApiResponse) {
+  const { longUrl } = req.query;
+  try {
+    const response = await fetch("https://api.tinyurl.com/create", {
+      method: "POST",
+      headers: {
+        accept: "application/json",
+        authorization: `Bearer ${API_KEY}`,
+        "content-type": `application/json`,
+      },
+      body: JSON.stringify({ url: longUrl }),
+    });
+
+    if (response.status !== 200) {
+      throw new Error(
+        `There was a problem with the fetch operation. Status Code: ${response.status}`
+      );
+    }
+
+    const responseData = await response.json();
+    const shortUrl = responseData.data.tiny_url;
+
+    if (!shortUrl) {
+      throw new Error(`There was a problem with fetching the shortUrl.`);
+    }
+
+    return res.status(200).json({ shortUrl });
+  } catch (error) {
+    return res.status(500).json({ error: error.message });
+  }
+}
```

Comment:
Waarom is dit in next.js gemaakt ipv Django? Ik had hier nog op gewezen in https://github.com/ZEnMo/Holon-webapp/issues/786#issuecomment-1604080587

Het is enigszinds mogelijk een static site te exporteren met next.js. TinyUrl zal niet werken op die static site. 

Geen blocker maar wel een beetje jammer.

Reply:
@Erikvv Ik had het even met Mattijs overlegd en omdat we al wel deze map hadden in de code leek het ons logisch om het in de bestaande structuur te schuiven. Maar dat zie jij anders? Anders even bellen via teams wat het handigste is om te doen? Ik kan de hele dag behalve tussen 14 - 15.

Reply:
Laat het maar zoals het nu is gemaakt, is goed genoeg.

Reply:
Ok, zou jij dan willen reviewen?